src/certified_datapacks/requirement.md

```diff
@@ -1,11 +1,27 @@
-# Requirement
-
-A datapack must satisfy these requirements to receive the Certified Datapack title
-
-- Datapack must be portable, it must not be tied down to a specific save.
-- Datapack must follow all of **Tier 3** Conventions.
-
-## Note
-
-1. We may rule a datapack out at our discretion, for reasons we would then specify.
-2. This requirement is not final and can be changed in the future.
+# Requirement
+
+A datapack must satisfy these requirements to receive the Certified Datapack title
+
+- Datapack must be portable, it must not be tied down to a specific save.
+- Datapack must follow all of **Tier 3** Conventions.
+- Use `core`, `library`, `API`, `utils`, etc. These datapacks must also be certified.
+  - Your review status will be paused until those datapacks are also certified.
+- If you are not the owner, **don't `steal`, `copy`, or `claim`** that you created it.
+  - `Modification` or `further develop/expand/build on` should first check the **license** or **contact the owner**, and remember to **credit**.
+
+## Note
+
+1. We may rule a datapack out at our discretion, for reasons we would then specify.
+2. This requirement is not final and can be changed in the future.
+
+### Note from Convention Mod
+
+- **Learn more about the convention** by reading the `Note from Convention Mod (Optional)` section, this allows you to confidently inspect your datapack before requesting review and reduces the Datapack Reviewer's workload.
+- **Request review:** please include every website where your datapack is published. It doesn't have to be just `GitHub` or `Planetminecraft`. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
+- **Review type:** please also specify the type of review you require.
+  -  **Certified Compatibility ✅ Only**
+  -  **Certified Compatibility ✅ + Style Guideline Followed 📚 (Optional)**
+- **Datapack unreleased:** please don't DM the file; instead, use `GitHub private repo` and invite the reviewer. Don't worry about any datapack functions leaking.
+- **After certified:** then your datapack will have an updated version, which you can request to review as many times as you want.
+- **Request review again:** please add the `changelog` or the `location of the changed file`. And If you're using `GitHub`, we can easily check the `commit history`. By doing so, the reviewer can't waste time reviewing what they've already reviewed.
+- If you have any questions about convention, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```

Comment:
Again, excessive use of bullet points and code blocks. Try to make the sentence flow naturally in paragraphs.

Reply:
```suggestion
- **Learn more about the convention** by reading the "Additional Guideline (Optional)" section, this allows you to confidently inspect your datapack before requesting review and reduces the Datapack Reviewer's workload.
- **Request review:** please include every website where your datapack is published. It doesn't have to be just GitHub or Planetminecraft. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
  - **Review type:** please also specify the type of review you require.
    -  **Certified Compatibility ✅ Only**
    -  **Certified Compatibility ✅ + Style Guideline Followed 📚 (Optional)** 
  - **After certified:** then your datapack will have an updated version, which you can request to review as many times as you want.
  - **Request again:** please add the *changelog* or the *location of the changed file*. And If you're using GitHub, we can easily check the commit history. By doing so, the reviewer can't waste time reviewing what they've already reviewed.
  - **Datapack unreleased:** please don't DM the file; instead, use *GitHub private repo* and invite the reviewer. Don't worry about any datapack functions leaking.
- If you have any questions about convention, you can ask on GitHub or mention `@Convention Mod` and `@Datapack Reviewer` on the discord server.
```

Reply:
No improvement from requested changes.